src/renderers/shared/server/ReactPartialRenderer.js

```diff
@@ -305,7 +305,6 @@ function createOpenTagMarkup(
   if (isRootElement) {
     ret += ' ' + DOMMarkupOperations.createMarkupForRoot();
   }
-  ret += ' ' + DOMMarkupOperations.createMarkupForID('');
```

Comment:
If we remove this then we break existing usage of ReactDOM.render and have no way of showing the warning, no?